File: benchmarks/fib/fib.js
function fib(n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

const n = parseInt(process.argv[2]) || 35;

const start = performance.now();
const result = fib(n);
const elapsed = performance.now() - start;
console.error(`TIME_MS:${elapsed.toFixed(2)}`);
console.log(result);

File: benchmarks/fib/fib.rb
def fib(n)
  return n if n <= 1
  fib(n - 1) + fib(n - 2)
end

n = ARGV[0] ? ARGV[0].to_i : 35

start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
result = fib(n)
elapsed = (Process.clock_gettime(Process::CLOCK_MONOTONIC) - start) * 1000

STDERR.puts "TIME_MS:#{format('%.2f', elapsed)}"
puts result
